tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py35, py36, py37, py38, pypy, pypy3
+envlist = security,flake8,pylint,docs,py
```

Comment:
I think it is nice to have a list of supported envs here, i.e. keep py36, py37, py38, pypy3, and add py39. What do you think about discussing this change separately? We had the same discussion for other repos.